Component-assets/Css/component-Responsive.css

```diff
@@ -59,4 +59,10 @@
   .sample__code{
     max-width:20rem;
   }
-}
+  .grid-two-container {
```

Comment:
good to see you are maintaining responsiveness of components

Reply:
Thankyou